src/checkbox.tsx

```diff
@@ -106,6 +106,17 @@ const useStyles = createUseStyles(() => ({
         opacity: 0.5,
```

Comment:
This screenshot is failing (note the checked vs unchecked states), do you know why?

![](https://camo.githubusercontent.com/6c4dd74029ad3b787320149d61828e3e81ab4c3fac243d0db3a976527ef3830f/68747470733a2f2f6465766d6973746963617765622e626c6f622e636f72652e77696e646f77732e6e65742f6d6973746963612d7765622d313633393033353233303638352f636865636b626f782d73637265656e73686f742d746573742d7473782d636865636b626f782d322d646966662e706e67)